components/toast/toast.css

```diff
@@ -0,0 +1,62 @@
+.toast {
+  padding: 0.5rem 1rem;
+  border-radius: 0.5rem;
+  color: var(--white-color);
+  margin: 1rem;
+  position: fixed;
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  min-width: 250px;
+  transition: all 0.5s ease-in-out;
+}
+.toast.success {
+  background: linear-gradient(270deg, var(--green-bg-color));
+}
+.toast.danger {
+  background: linear-gradient(45deg, var(--red-bg-color));
+}
+.toast.loading {
+  background: linear-gradient(45deg, var(--blue-bg-color));
+}
+
+.show-mode {
+  position: static;
+  transform: translateX(0%);
+}
+
+.hidden {
+  visibility: hidden;
+}
+
+/* snackbar */
+
+.snackbar {
+  width: 60%;
+  padding: 1rem;
+  background-color: #212121;
```

Comment:
using variable here will be good

Reply:
I'll move this colour value into a variable. Thank you for pointing that out.